src/organisms/pricing/PricingPlan.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import Row from '../../atoms/row/Row';
+import PricingItem from '../../molecules/pricing/PricingItem';
+import TabAction from '../../molecules/tabAction/TabAction';
+import styles from './PricingPlan.module.css';
+
+const PricingPlan = () => {
+  const [isSelected, setisSelected] = useState(1);
+  useEffect(() => {
+    console.log(isSelected);
+  }, [isSelected]);
+  const pricing = [
+    {
+      name: 'Regular',
+      subName: 'Starter plan',
+      features: [
+        { name: 'Limited Projects', isAvailable: true },
+        { name: 'Regular Support Business', isAvailable: true },
+        { name: '1 Month Free Trial', isAvailable: true },
+        { name: '3GB Storage', isAvailable: true },
+        { name: 'Ads Preview', isAvailable: true }
+      ],
+      price: 'Free'
+    },
+    {
+      name: 'Platinum',
+      subName: 'For the best results',
+      features: [
+        { name: 'Limited Projects', isAvailable: true },
+        { name: 'Regular Support Business', isAvailable: true },
+        { name: '1 Month Free Trial', isAvailable: true },
+        { name: '20GB Storage', isAvailable: true },
+        { name: 'Ads Preview', isAvailable: true }
+      ],
+      price: '$324'
+    },
+    {
+      name: 'Standard',
+      subName: 'Most popular',
+      features: [
+        { name: 'Limited Projects', isAvailable: true },
+        { name: 'Regular Support Business', isAvailable: true },
+        { name: '1 Month Free Trial', isAvailable: true },
+        { name: '10GB Storage', isAvailable: true },
+        { name: 'Ads Preview', isAvailable: true }
+      ],
+      price: '$234'
+    }
+  ];
+  // const PricingItemClickHandler = (data) => {
```

Comment:
Here too. Commented code most likely should not be there at all. If it is a TODO kind of thing, maybe this could be added to a JIRA task instead.